signin.js

```diff
@@ -0,0 +1,59 @@
+/*
+4주차 주요 추가 사항: sign.js
+이전 코드는 코드잇에서 제공한 3주차 템플릿 사용
+
+세부 사항
+이메일과 비밀번호 입력 필드의 유효성 검사.
+올바른 로그인 정보 입력 시 페이지 이동.
+에러 메시지 표시 및 디자인 적용.
+로그인 실행 시 로그인 버튼 클릭 또는 Enter키 입력 인식.
+비밀번호 가시성 토글 및 아이콘 변경.
+*/
+
+document.addEventListener("DOMContentLoaded", function() {
+  const emailInput = document.getElementById("email-input");
+  const passwordInput = document.getElementById("password-input");
+  const emailError = document.getElementById("email-error");
+  const passwordError = document.getElementById("password-error");
+  const togglePassword = document.getElementById("toggle-password");
+  const eyeIcon = document.getElementById("eye-icon");
+
+  emailInput.addEventListener("focusout", function() {
```

Comment:
```suggestion
  emailInput.addEventListener("focusout", () => {
```

익명 함수로 작성했을때 코드 실행시 큰 문제점은 없습니다.(function 키워드의 this문제 제외) 하지만 익명 함수로 쓸거라면 차라리 화살표 함수를 사용하시거나 기명 함수를 파라미터로 전달하는 방식을 권장드립니다.

- [익명 함수 관련 참고자료](https://daklee.tistory.com/entry/%EC%9D%B5%EB%AA%85-%ED%95%A8%EC%88%98Anonymouse-Function%EC%97%90-%EB%8C%80%ED%95%98%EC%97%AC)